portfolio/src/main/webapp/styles/article-text.css

```diff
@@ -0,0 +1,13 @@
+h2, p{
+     margin-bottom: 2.4rem;
+ }
+ h1  { font: 700 2.0rem/1.2 "Roboto"; color: hsl(000, 0%, 33%);}
```

Comment:
I'm not going to hold up the pull request over this, but please note for future work that styling for CSS is inconsistent across this PR. Sometimes there's a space before an opening `{`, sometimes there's not, sometimes there are several. Same thing for closing braces. Using a consistent style helps readers (reviewer, future maintainers) understand your code more easily -- as engineers, we're trained to notice patterns and deviations from them, so these inconsistencies stand out and distract from the intent of the code.

Reply:
Yes, that does make sense, I'll make the code a lot more consistent with respect to a coding style.